context/Dockerfile

```diff
@@ -1,46 +1,9 @@
-FROM ubuntu:14.04
+FROM magfest/sideboard
 MAINTAINER Dominic Cerquetti "dom@magfest.org"
 
-# TODO: probably want to copy this stuff in here:
-# https://github.com/magfest/ubersystem-deploy/blob/master/vagrant/vagrant.sh
-
-# base system stuff
-RUN apt-get -y update && apt-get -y install git python3 python3-gdbm python3-tk gcc python3-dev libpq-dev wamerican language-pack-id && apt-get clean
-
-RUN locale-gen en_US en_US.UTF-8 hu_HU hu_HU.UTF-8
-RUN dpkg-reconfigure locales
-
-# there is a bug in python -m venv which doesnt obey --copies 
-# (i.e. don't do any symlinks). We have submitted this to CPython 
-# for inclusion but for now, patch the installed python file.
-# (we can't use symlinks with SMB shares).  May not be strictly necessary
-# for docker in production, but, doesn't hurt to have this fix.
-ADD venv-symlink-fix.patch /tmp/venv-symlink-fix.patch
-RUN patch /usr/lib/python3.4/venv/__init__.py < /tmp/venv-symlink-fix.patch
-
```

Comment:
was this patch fixed in ubuntu's 3.4?  cause if it wasn't this patch isn't needed for linux host running vagrant, but it is needed for windows host running vagrant.